### PR TITLE
Fix gas price explanation

### PR DESCRIPTION
**Pull request by gregnazario**
### Checklist

- Do all Lints pass?
  - [x] Have you ran `pnpm spellcheck`?
  - [x] Have you ran `pnpm fmt`?
  - [x] Have you ran `pnpm lint`?

### CODE REVIEW

File: apps/nextra/pages/en/network/blockchain/gas-txn-fee.mdx [hunk 1]
```diff
@@ -90,13 +90,13 @@ However, within a block, the order of transaction execution is determined by the
 
 When a transaction is submitted to the Aptos blockchain, the transaction must contain the following mandatory gas fields:
 
-- `max_gas_amount`: The maximum number of gas units that the transaction sender is willing to spend to execute the transaction. This determines the maximum computational resources that can be consumed by the transaction.
-- `gas_price`: The gas price the transaction sender is willing to pay. It is expressed in Octa units, where 1 Octa equals 10<sup>-8</sup> Aptos utility token.
+- `max_gas_amount`: The maximum number of gas units that the transaction sender is willing to spend to execute the transaction. This determines the maximum computational and storage resources that can be consumed by the transaction.
```
Comment (hariria) [inline apps/nextra/pages/en/network/blockchain/gas-txn-fee.mdx:93]: Is IO also included in gas calculations? Or is that just derived from storage

File: apps/nextra/pages/en/network/blockchain/gas-txn-fee.mdx [hunk 2]
```diff
@@ -90,13 +90,13 @@ However, within a block, the order of transaction execution is determined by the
...
+- `gas_price`: The price per gas unit the transaction sender is willing to pay. It is expressed in Octa units, where 1 Octa equals 10<sup>-8</sup> Aptos utility token.
```
Comment (hariria) [inline apps/nextra/pages/en/network/blockchain/gas-txn-fee.mdx:94]: 10^8 Octas is 1 APT might be clearer

**Review by hariria — APPROVED** [review]

File: apps/nextra/pages/en/network/blockchain/gas-txn-fee.mdx (hunk 1 above)
Comment (gregnazario) [reply apps/nextra/pages/en/network/blockchain/gas-txn-fee.mdx:93]: Storage = IO

File: apps/nextra/pages/en/network/blockchain/gas-txn-fee.mdx (hunk 2 above)
Comment (gregnazario) [reply apps/nextra/pages/en/network/blockchain/gas-txn-fee.mdx:94]: I don't know who wrote this bad description, I don't remember writing Aptos utility token...

File: apps/docusaurus/docs/concepts/blockchain.md [hunk 3]
```diff
@@ -68,7 +68,7 @@ The raw transaction includes the following fields:
...
+| [Maximum gas amount](../reference/glossary.md#maximum-gas-amount)                    | The maximum gas amount in Aptos utility tokens Alice is willing to pay for this transaction. Gas charges are equal to the base gas cost covered by computation and IO multiplied by the gas price. Gas costs also include storage with an Apt-fixed priced storage model. This is represented as Octa or units of 10<sup>-8</sup> Aptos utility tokens.                                                                                                                                                                                                                                |
```
Comment (hariria) [inline apps/docusaurus/docs/concepts/blockchain.md:71]: Aptos utility token is mentioned

**Review by hariria — APPROVED** [review]

**Review by hariria — APPROVED** [review]

File: apps/docusaurus/docs/concepts/blockchain.md (hunk 3 above)
Comment (gregnazario) [reply apps/docusaurus/docs/concepts/blockchain.md:71]: Oh, this is docusaurus, so I'm not bothering.

**Pull request merged**